domains/grid/utils/gridParser.ts

```diff
@@ -196,3 +198,22 @@ const parseInstagramWidgetInput = (
 
   throw new Error('Invalid Instagram input')
 }
+
+const parseElfSightWidgetInput = (
+  input: string
+): GridWidgetExtended | never => {
+  const ELFSIGHT_URL_REGEX = /class="elfsight-app-([\w-]+)"/
+
+  const [, id] = input.match(ELFSIGHT_URL_REGEX) || []
+
+  if (id) {
+    return {
+      type: GRID_WIDGET_TYPE.enum.ELFSIGHT,
+      properties: {
+        id: id,
```

Comment:
The properties doesn't match with schema, you have `id` here and schema has `src`